code/items/ammo/BuckshotAmmo.cs

```diff
@@ -0,0 +1,13 @@
+using Sandbox;
+
+namespace TTTReborn.Items
+{
+    [Library("ttt_ammo_buckshot")]
+    partial class BuckshotAmmo : TTTAmmo
+    {
+        public override string AmmoType => "buckshot";
```

Comment:
Can this be an enum now? I might have missed some of the convo on the `string` discussion.

Reply:
It cannot.

The goal with using `string` rather than an enum allows for expandability. Meaning additional ammo types (especially those coming from custom addons) will easily be recognized and placed into the ammo dictionary within your ammo inventory. If it returns to being an `enum`, new ammo types cannot be added without editing the enum within the main TTTReborn gamemode's source. Having to manage all that and keep it up to date is an unnecessary piece of maintenance to add for future folk.

Reply:
Yeah kk that's what I assumed, thanks. I haven't really written any of the `extensible` stuff.